docs/source/index.rst

```diff
@@ -0,0 +1,25 @@
+Documentation
+=============
+
+Executor
+---------
+The primary object in pywren is an `executor`. The standard way to get everything set up is to import pywren, and call the `default_executor` function.
+
+.. automethod:: pywren.wren.default_executor
+
+`default_executor()` reads your `pywren_config` and returns `executor` object that's ready to go.
+We can use this `executor` to run `map`, which applies a function to a list of data in the cloud.
+
+.. automethod:: pywren.executor.Executor.map
+
+
+Waiting for the results
+--------------------------
+
+.. autofunction:: pywren.wait.wait
```

Comment:
again, as we import all functions into top APIs for pywren, we can just do `pywren.wait`

Reply:
Was this fixed ?

Reply:
Yes. `pywren.wait.wait` is a pointer to  the module.file.function, and it shows up simply as `wait` in the docs

http://pywren.io/docs/